### ISSUE TITLE
Dropdown Input:  Single and multi-select open state

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#695]
**Issue:**
1. SR reads the search bar even though it is not active

2. Reads 'menu popup combo box' which is misleading because it does not mention it's a single-select or multi-select menu.

3. No focus control indicator on options within dropdown menu.

**Solution:**
1. SR should not read the search bar when it is not present in the component.

2. SR should read that it is either a single-select or multi-select menu.

3. Add focus control indicators to dropdown menu.
